ocf/serviced-storage

```diff
@@ -121,6 +132,18 @@ END
 }
 
 ##############################################################################
+# Get the value of a serviced configuratoin property. Return empty string if
+# the property is not found.
+get_serviced_property() {
```

Comment:
You may want to call serviced to get the configuration info instead of trying to rewrite a config parser.